tarodb/assets_store_test.go

```diff
@@ -535,6 +557,8 @@ func (a *assetGenerator) genAssets(t *testing.T, assetStore *AssetStore,
 
 	ctx := context.Background()
 	for _, desc := range assetDescs {
+		desc := desc
```

Comment:
LOL, I also don't trust loop variables anymore either :joy: Too many scars...

Reply:
Heh, yeh I just have an instinct to swap it in now whenever I'm scanning a call site looking for bugs 😆